src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Rectangle implements Figure {
```

Comment:
The class `Rectangle` should not begin with an empty line. Please remove the empty line at the start of class definition.